connect/transforms/src/test/java/org/apache/kafka/connect/transforms/InsertFieldTest.java

```diff
@@ -104,11 +104,64 @@ public void schemalessInsertConfiguredFields() {
 
         final SourceRecord transformedRecord = xform.apply(record);
 
-        assertEquals(42L, ((Map) transformedRecord.value()).get("magic"));
-        assertEquals("test", ((Map) transformedRecord.value()).get("topic_field"));
-        assertEquals(0, ((Map) transformedRecord.value()).get("partition_field"));
-        assertEquals(null, ((Map) transformedRecord.value()).get("timestamp_field"));
-        assertEquals("my-instance-id", ((Map) transformedRecord.value()).get("instance_id"));
+        assertEquals(42L, ((Map<?, ?>) transformedRecord.value()).get("magic"));
+        assertEquals("test", ((Map<?, ?>) transformedRecord.value()).get("topic_field"));
+        assertEquals(0, ((Map<?, ?>) transformedRecord.value()).get("partition_field"));
+        assertEquals(null, ((Map<?, ?>) transformedRecord.value()).get("timestamp_field"));
+        assertEquals("my-instance-id", ((Map<?, ?>) transformedRecord.value()).get("instance_id"));
+    }
+
+    @Test
+    public void insertConfiguredFieldsIntoTombstoneEvent() {
```

Comment:
Nit: might be a bit clearer to add "WithoutSchema":
```suggestion
    public void insertConfiguredFieldsIntoTombstoneEventWithoutSchema() {
```

Reply:
Done.